config/plugins/visualizations/graphviz/templates/graphviz.mako

```diff
@@ -10,9 +10,8 @@
         root = h.url_for( '/' )
     %>
 
-    <script type="text/javascript" src="/static/scripts/libs/jquery/jquery.js"></script>
-
     ${h.stylesheet_link( root + 'static/plugins/visualizations/graphviz/static/css/style.css' )}
+    ${h.javascript_link( root + 'static/plugins/visualizations/graphviz/static/js/jquery.min.js' )}
```

Comment:
Could we use the jquery we ship already (`static/scripts/libs/jquery.custom.js`) instead of another copy that this commit includes?

Reply:
ping @guerler

Reply:
It might be safer to use the jQuery shipped with this visualization in order to avoid version conflicts, particularly since we are in the process of removing the jQuery provided by Galaxy. ping @dannon.